services/prefect/flows/utils.py

```diff
@@ -1,73 +0,0 @@
-import prefect
-from prefect.client import Client
-from prefect.engine.state import Skipped, State
-from prefect import Flow
-from typing import Optional, Set
-
-
-def skip_if_running_handler(obj, old_state, new_state):
```

Comment:
No longer using these state handlers for simplicities sake. This means we won't be able to immediately tell when a flow has been "skipped" without looking at the task runs, which I think is fine